lib/ransack_ui/ransack_overrides/helpers/form_builder.rb

```diff
@@ -167,7 +167,7 @@ def attribute_collection_for_base(base, attributes=nil)
             }.to_json
           end
 
-          if foreign_klass && (autocomplete_source = foreign_klass.constantize._ransack_autocompletes_through rescue false)
+          if foreign_klass && (autocomplete_source = foreign_klass._ransack_autocompletes_through rescue false)
```

Comment:
Maybe pull this long `()` into its own line please, it's a bit much for an inline assignment on `if`:

`if autocomplete_source` ...

rescue should be `nil` too (like a `try`).